sixtyfps_runtime/interpreter/eval.rs

```diff
@@ -1079,3 +1097,16 @@ pub fn default_value_for_type(ty: &Type) -> Value {
         }
     }
 }
+
+lazy_static::lazy_static! {
+    static ref INTERNED_STRINGS: std::sync::Mutex<std::collections::HashSet<String>> = Default::default();
+}
+
+pub fn interned_str(s: &str) -> &'static str {
```

Comment:
Where is that used? I don't see the reference to it.

Is there maybe a crate we can use?